samples/tfm/tfm_cert/prj.conf

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
+
+CONFIG_TFM_LOG_LEVEL_SILENCE=n
+CONFIG_TFM_SPM_LOG_LEVEL_DEBUG=y
+CONFIG_TFM_EXCEPTION_INFO_DUMP=y
+
+CONFIG_SECURE_BOOT=y
+CONFIG_BUILD_S1_VARIANT=n
+
+CONFIG_FW_INFO=y
+CONFIG_FW_INFO_FIRMWARE_VERSION=1
```

Comment:
Maybe I'm wrong but this should set the version for NISB, so it should be set for the MCUboot image, as the MCUboo image is the one that gets booted and "upgraded" by NISB. 
And for the application we should set the MCUBoot_version